### DISCUSSION TITLE
Print to local printer from workflow task

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#9718] [Q&A]
Hello,

I was wondering if it's possible to print from my web server to a local POS printer.

My dad has a webshop built in OrchardCore. He would like to receive order tickets when a order is completed so I created an Workflow event that will be triggered when an order is clompleted and an Workflow task that will print te ticket to his printer.

The only problem is that the printer is not reachable from the web server because it's not in the same network. Does anyone know if this is possible to do or do I have to find an other way to do this?

Thanks!

**Comment by hishamco** [comment]
This is not OC related, basically set your printer as default one, then if you pressed Ctrl+P and any web page is printed, you can easily then print ~~order bill~~ ticket

**Comment by ns8482e** [comment]
Use Printer that support web/eprint and using OC workflow send an email to your printer address.

**Reply by Skrypt** [reply]
Unless the printer is connected on the same network locally or connected physically to the server ; you will need to do what @ns8482e suggests.

From C# (server side) you could also trigger https://github.com/wkhtmltopdf/wkhtmltopdf to create a PDF of your order page and then print it where you want. This is what they do for example in Odoo.
